docs/how-to-guides/feathr-configuration-and-env.md

```diff
@@ -85,6 +85,12 @@ Feathr will get the configurations in the following order:
 | FEATURE_REGISTRY__PURVIEW__TYPE_SYSTEM_INITIALIZATION (Deprecated Soon) | Controls whether the type system (think this as the "schema" for the registry) will be initialized or not. Usually this is only required to be set to `True` to initialize schema, and then you can set it to `False` to shorten the initialization time.  | Required if using Purview directly without registry service. Deprecate soon, see [here](#deprecation) for more details. |
 | MAVEN_ARTIFACT_VERSION                                                  | Version number like `0.9.0`. Used to define maven package version when main jar is not defined.                                                                                                                                                            | Optional                                                                                                                |
 
+# A list of environment variables that Feathr uses when running registry and service
+| Environment Variable                                                    | Description                                                                                                                                                                                                                                                | Required?                                                                                                               |
+| ----------------------------------------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------------------------------------------------------------------------------------------------------------------- |
+| FEATHR_SANDBOX                                          | If it is set to any value, the registry server will be running in sandbox mode and will connect to a local database with SQLite.                                                                                                                                                                 | Optional                                                                                                                |
```

Comment:
I'm not sure if it's a good idea to couple FEATHR_SANDBOX with SQLite. It might be better to be configurable, which enables us to extend it other on premise database.

Reply:
Agree. I think eventually we should move to a more generic ORM based approach and use SQL Alchemy so that multiple backend will be supported. After this PR is in, let's focus on that part to make it configurable (and maybe deprecate the pure SQL approach)